test/example.c

```diff
@@ -7,6 +7,19 @@
 
 #include "zlib.h"
 #include <stdio.h>
+#ifndef UINT32_MAX
+#include <stdint.h>
+#endif
+
+/*support MSVC 2013 prior to https://github.com/libgit2/libgit2/issues/3476 */
+#if defined(_MSC_VER)
+ GIT_BEGIN_DECL
+# include "inttypes.h"
+ GIT_END_DECL
+/** This check is needed for importing this file in an iOS/OS X framework throws an error in Xcode otherwise.*/
```

Comment:
Comment format (this and the rest) is /* text */ specifically space before and after, don't start with a capital